packages/paste-website/src/pages/experiences/console/index.mdx

```diff
@@ -0,0 +1,379 @@
+evolve to help the user understand the feature and/or start using it. You can use an [empty state](/patterns/empty-state) with an
+illustration and information about the feature.
+
+Hiding a page in the nav (via an account flag) should be done only when you are limiting access to that feature.
+For example, the feature is in private beta or the feature is deprecated and supported only for existing users.
+Hiding and showing pages can also be done when you’re limiting access based on RBAC.
+
+## Beta tags
+
+Beta should appear in the nav under the product name when a product is in beta. It can also appear under a
+single page in a product nav when just that feature is in beta. In addition, you should add the beta badge
+next to the H1 in the page for the feature in beta. In the case where the entire product is in beta,
+include the badge next to the H1 on the first page in the nav. The “beta” is clickable and opens a modal
+describing what beta means.
+
+![](../../../assets/images/experiences/console/beta-tag.png)
+
+<ImageCaption>A beta tag being used in Console to identify and describe a beta-stage product.</ImageCaption>
+
+---
+
+## Product Explorer
+
+The Product Explorer is designed to be a catalog of Twilio products each with a short description,
+a link to the first page in the product in Console, and an external link to docs for that product
+(twilio.com/docs/&lt;product&gt;). This page provides a short description of what a product does, shows if the
+product is available in more than the US1 region, and allows you to pin them to the left navigation.
+
+The Product Explorer shows one card per product, regardless of how many different offerings exist in your
+product. There is a one-to-one relationship between the product card and the product that gets pinned to
+the left navigation.
+
+When introducing a new product to the Console, work closely with the Console team to determine the
+appropriate category to include your product in (testing may be required).
+
+Product description should be between 35-60 characters to fit (see checklist below).
+
+Products that appear in the explorer page should be products that you develop with and relate to your
+Twilio application. If you have questions about whether your product should be included here,
+reach out to the Console team.
+
+![](../../../assets/images/experiences/console/product-explorer.png)
+
+<ImageCaption>A screenshot of the Product Explorer page showing a catalog of Twilio products by category</ImageCaption>
+
+---
+
+## Search ("Jump to...")
+
+The "jump to" bar in the Console masthead area allows users to quickly jump to a specific page in the Console.
+The results will automatically populate with your published product pages. It is recommended that you also add
+keywords to help with findability of your product pages.
+
+- The JSON data driving search is located here: https://code.hq.twilio.com/twilio/console-navigation-backend/blob/main/json/consoleSearch.json
```

Comment:
Showing this full URL is a little messy, and probably not super accessible. I wonder if we could re-word this whole section to be a little bit more prose-y, maybe something like:

> The "jump to" bar in the Console masthead area allows users to quickly jump to a specific page in the Console.
The results will automatically populate with your published product pages. It is recommended that you also add keywords to help with findability of your product pages by [updating the search JSON](https://code.hq.twilio.com/twilio/console-navigation-backend/blob/main/json/consoleSearch.json) and creating a PR in the repository.

Reply:
oooo yes I like. Just updated it to this!